assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -106,6 +106,20 @@
 							#end
 						</a>
 					</th>
+					#if ($createdBy_enabled)
+						<th id="createdBy" scope="col">
+							<a href="javascript:void(0)" onclick="location='#toolLinkParam("$action" "doSort" "criteria=assignment_createdBy")'; return false;"   title="$tlang.getString("listassig.sorbycreator")">
```

Comment:
Would like to avoid declaring onclick handlers like this

Reply:
All others onclick handlers are like this. I think we would create a new JIRA and change all of them.

Reply:
We have been discusing this here and we think this must be modified all at once because there is a JS modification and if you do this bit by bit you would have two different behaviour for the "same" column table order function.